src/lib/stitching/exchange/stitching.ts

```diff
@@ -252,6 +256,28 @@ export const exchangeStitchingEnvironment = ({
         ...totalsResolvers("CommerceOffer", offerAmountFields),
         fromDetails: fromDetailsResolver,
       },
+      Me: {
+        orders: {
+          fragment: gql`... on Me {
+            __typename
+          }`,
+          resolve: async (_source, args, context, info) => {
+            console.log("------> here", args, _source)
+            const result = await info.mergeInfo.delegateToSchema({
+              schema: exchangeSchema,
+              operation: "query",
+              fieldName: "myOrders",
```

Comment:
I think you're running into an issue here, since you're actually removing it via transform. Following the stitching logic in https://github.com/artsy/metaphysics/blob/9a0e9f30707964b5bd7c853a0c9ccc9fbb899f93/src/lib/stitching/mergeSchemas.ts , the transforms are run over `exchangeSchema` prior to merging in.

So here, the error is because `myOrders` isn't found!

I think the fix is to _drop_ the transform that removes it (and then refer to it here by the transformed name, `commerceMyOrders`), or some such combination. I think that will work.

Reply:
yeah! that worked! thanks!!!

now next question is how to remove this `commerceMyOrders` from root fields of our global schema?

Reply:
something like [this](https://github.com/artsy/metaphysics/blob/master/src/lib/stitching/vortex/schema.ts#L23-L35)? But can we remove it now or deprecate the root query, update reaction and then get rid of it?